apps/emqx_conf/src/emqx_conf_cli.erl

```diff
@@ -118,6 +135,21 @@ admins(["fast_forward", Node0, ToTnxId]) ->
 admins(_) ->
     emqx_ctl:usage(usage_sync()).
 
+fix_inconsistent_with_raw(Node, Keys) ->
+    Confs = lists:foldl(
+        fun(Key, Acc) ->
+            KeyRaw = atom_to_binary(Key),
+            Acc#{KeyRaw => emqx_conf_proto_v4:get_raw_config(Node, [Key])}
+        end,
+        #{},
+        Keys
+    ),
+    ok = emqx_cluster_rpc:reset(),
```

Comment:
Erasing all history sounds a bit risky IMO.
Can we try to fast-forward the target node(s), then load?